pkgs/servers/uwsgi/default.nix

```diff
@@ -0,0 +1,76 @@
+{ stdenv, lib, fetchurl, pkgconfig, jansson
+, plugins
+, pam, withPAM ? stdenv.isLinux
+, systemd, withSystemd ? stdenv.isLinux
+, python2, python3, ncurses
+}:
+
+let pythonPlugin = pkg : { name = "python${if pkg ? isPy2 then "2" else "3"}";
+                           interpreter = pkg;
+                           path = "plugins/python";
+                           deps = [ pkg ncurses ];
+                           install = ''
+                             install -Dm644 uwsgidecorators.py $out/${pkg.sitePackages}/uwsgidecorators.py
+                             ${pkg.executable} -m compileall $out/${pkg.sitePackages}/
+                             ${pkg.executable} -O -m compileall $out/${pkg.sitePackages}/
+                           '';
+                         };
+    available = [ (pythonPlugin python2)
+                  (pythonPlugin python3)
+                ];
+     needed = builtins.filter (x: lib.any (y: x.name == y) plugins) available;
```

Comment:
The indention of this line is more than of the other lines. Is there any reason for this? 

Sorry for bitching around because of this, but this stuff makes reviewing hard, at least for me... :smirk: